docs/package/gluon-config-mode-geo-location.rst

```diff
@@ -35,8 +35,8 @@ config_mode.geo_location.map_lat \: optional
   - represents the default latitude center of the location picker map.
   - defaults to ``0.0``
 
-The above 2 options will usually shown on a factory flashed Router. If a node
-is reenter the config mode the maps center will be on the last defined
+The above 2 options will usually be shown on a factory flashed Router. If a node
+is reentered the config mode the map's center will be on the last defined
```

Comment:
"a node is reentered" is still wrong, i'm not sure about the ideal way to say it... @mweinelt ?